src/Plugin/pathauto/AliasType/ForumAliasType.php

```diff
@@ -57,4 +57,16 @@ public function getSourcePrefix() {
     return 'forum/';
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function applies($object) {
+    if (parent::applies($object)) {
```

Comment:
does the parent call really work here? ForumAliasType won't have a derivative ID, so that will be an empty string? Probably just hardcode instance TermInterface?